### PR TITLE
Kaldb zipkin api

### PR DESCRIPTION
**Pull request by shellywu815**
Added the skeleton code for functions in ZipkinService.java and added a zipkinService instance into Kaldb.java

### CODE REVIEW

File: kaldb/src/main/java/com/slack/kaldb/zipkinApi/ZipkinService.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            @Param("limit") @Default("10") Integer limit)
+            throws IOException {
+        return null;
+    }
+
+    @Get("/api/v2/trace/{traceId}")
+    public HttpResponse getTraceByTraceId(@Param("traceId") String traceId) throws IOException {
+        return null;
+    }
+}
```
Comment (vthacker) [inline kaldb/src/main/java/com/slack/kaldb/zipkinApi/ZipkinService.java:57]: maybe add a newline at the end?

**Comment by vthacker** [comment]
Nice! This is a good start. Can you make two changes to this
1. Make any of the 4 functions that you have implemented not return null, but return `httpresponse(OK,string)` where the string can be `[]` 
2. Add a test like ElasticsearchApiServiceTest to verify if you are able to query ZipkinService

File: kaldb/src/main/java/com/slack/kaldb/zipkinApi/ZipkinService.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.server.annotation.*;
+import com.slack.kaldb.server.KaldbQueryServiceBase;
+import java.io.IOException;
+import java.util.*;
+
+@SuppressWarnings("OptionalUsedAsFieldOrParameterType")
+public class ZipkinService {
```
Comment (bryanlb) [inline kaldb/src/main/java/com/slack/kaldb/zipkinApi/ZipkinService.java:12]: We should have a comment here that explains what the purpose of the class is, sorta like https://github.com/slackhq/kaldb/blob/master/kaldb/src/main/java/com/slack/kaldb/elasticsearchApi/ElasticsearchApiService.java#L37 or https://github.com/slackhq/kaldb/blob/master/kaldb/src/main/java/com/slack/kaldb/clusterManager/RecoveryTaskAssignmentService.java#L36

**Review by bryanlb — APPROVED** [review]
Suggest adding a comment, other than that this looks fine 👍

**Pull request closed without merging**